website/cue/reference/components.cue

```diff
@@ -327,6 +328,8 @@ components: {
 
 	#FeaturesRoute: {}
 
+	#FeaturesExclusiveRoute: {}
```

Comment:
Do you know what this does? I can't actually find where this is used in the docs 🤔

Reply:
I had to add it otherwise `make generate-component-docs` fails. 

It cue so I didn't dive into it 🙈. If you scroll up there is a non-empty feature block: https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components.cue#L313